chapters/annotations.tex

```diff
@@ -1385,10 +1387,11 @@ \section{Annotations for the Graphical User Interface}\label{annotations-for-the
 The \lstinline!showStartAttribute = true! is primarily intended for non-parameter values and avoids introducing a separate parameter for the start-value of the variable.
 \end{nonnormative}
 
-If \lstinline!colorSelector = true!, it indicates that an rgb-value selector can be presented for a vector of three elements and generate values 0..255 (the color selector should be useable both for vectors of \lstinline!Integer! and \lstinline!Real!).
+If \lstinline!colorSelector = true!, it suggests that an rgb-value selector can be presented for a vector of three elements and generate values 0..255 (the color selector should be useable both for vectors of \lstinline!Integer! and \lstinline!Real!).
```

Comment:
If we switch to suggesting things, I think we can be more direct:
```suggestion
If \lstinline!colorSelector = true!, it suggests use of an rgb-value selector to pick a vector of three elements with values in the range 0..255 (the color selector should be useable both for vectors of \lstinline!Integer! and \lstinline!Real!).
```

Reply:
I prefer the less direct current variant here. The reason is that from the user perspective it should be about selecting a color (which will generate an rgb-vector); not about picking a vector. Clearly it can be reformulated in other ways.